services/ui-src/src/components/alerts/ErrorAlert.tsx

```diff
@@ -26,10 +26,9 @@ export const ErrorAlert = ({
             status={AlertTypes.ERROR}
             title={error.title}
             description={error.description}
-            showIcon={false}
+            showIcon={showIcon}
             className={variant}
-            sx={sx.root}
-            {...props}
+            sxOverride={{ ...sx, ...sxOverride }}
```

Comment:
related to the above
```suggestion
            sxOverride={{ ...sx.root, ...sxOverride }}
```